src/js/formprint/index.js

```diff
@@ -0,0 +1,98 @@
+import 'js/base/setup';
+import 'js/i18n';
+
+import Backbone from 'backbone';
+import Radio from 'backbone.radio';
+import { View, CollectionView } from 'marionette';
+import hbs from 'handlebars-inline-precompile';
+
+import 'sass/provider-core.scss';
+import 'sass/app-root.scss';
+
+import './formprint.scss';
+
+import initPlatform from 'js/utils/platform';
+
+import App from 'js/base/app';
+
+import FormsService from 'js/services/forms';
+
+import 'js/entities-service';
+
+import BootstrapService from 'js/services/bootstrap';
+
+import IframeFormBehavior from 'js/behaviors/iframe-form';
+
+const RootView = CollectionView.extend({
+  viewComparator: false,
+  el: 'body',
+  template: hbs``,
+  initialize({ model }) {
+    this.render();
+
+    this.addChildView(new FormIframeView({ model }));
+  },
+});
+
+const FormIframeView = View.extend({
+  behaviors: [IframeFormBehavior],
+  className: 'form__frame',
+  template: hbs`
+    <div class="app-frame__content flex-region">
+      <div class="form__layout">
+        <div class="form__content">
+          <iframe src="/formapp/"></iframe>
+        </div>
+      </div>
+    </div>
+  `,
+});
+
+const FormPrintApp = App.extend({
```

Comment:
My guess is that in order to pull this off with the response, we'll need to pull what we need from the form service and add it to this app, removing the form service.  We have all of the requests we need from the three ids, but they don't cleanly align with what the current form service is doing.

Reply:
Hmm okay, I'll investigate getting this to work.